cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-labels.asciidoc

```diff
@@ -1,89 +0,0 @@
-=== Traversing the graph with multi-labeled nodes
```

Comment:
Content was moved to neo4j-manual-modeling/4.3/cypherManual/asciidoc/access-control/limitations.adoc